app/frontend/src/features/communities/CommunityInfoPage.tsx

```diff
@@ -0,0 +1,48 @@
+import { InfoIcon } from "components/Icons";
+import Markdown from "components/Markdown";
+import PageTitle from "components/PageTitle";
+import { Community } from "pb/communities_pb";
+import makeStyles from "utils/makeStyles";
+
+import CommunityLeadersSection from "./CommunityLeadersSection";
+import { GENERAL_INFORMATION } from "./constants";
+
+const useStyles = makeStyles((theme) => ({
+  title: {
+    alignItems: "center",
+    display: "flex",
+    "& > * + *": {
+      marginInlineStart: theme.spacing(1),
+    },
+  },
+  content: {
+    "& p": {
+      margin: 0,
```

Comment:
Why is this needed here and not the markdown component in general?

Reply:
I haven't checked if this will break in other places and if we want to disable the margin too, my guess is probably yes? I'll move it into `Markdown` if that's the case 👍